website/docs/dependencies/0031-install-cuda-and-nvidia-drivers.md

```diff
@@ -0,0 +1,111 @@
+
+| **Parameter** | **Required** | **Description**            |                 **Default**                     |
+|---------------|--------------|----------------------------|-------------------------------------------------|
+| PackageName   | Yes          | The logical name of the package that will be registered with the Virtual Client runtime. This name can be used by other profile components to reference the installation parent directory location for Drivers. |  |
+| Scenario      | No           | A name/identifier for the specific component in the profile. This is used for telemetry purposes only with components in dependency sections of the profile (i.e. cannot be used with --scenarios option on the command line). |  |
+| LinuxCudaVersion | Yes | The version of CUDA to be installed in Linux Systems |  |
+| LinuxDriverVersion | Yes | The version of Nvidia GPU driver to be installed in Linux Systems |  |
+| LinuxLocalRunFile | Yes | The link to local runfile to install Cuda and Nvidia GPU driver in Linux Systems | |
+| Username | No | The user who has the ssh identity registered for. | <Current UserName> |
+
+## Example
+The following sections provides examples for how to integrate the component into a profile.
+  
+### Windows example for downloading drivers from Web
+A sample URL for NVIDIA Drivers for Windows 10/11 is mentioned in example. The exact URL for the specific OS and Driver Version can be taken from NVIDIA Drivers website, given above.
+  <div class="code-section">
+
+  ```json
+  {
+      "Type": "WgetPackageInstallation",
+      "Parameters": {
+          "Scenario": "DownloadCudaAndNvidiaDriverUsingWget",
+          "PackageUri": "https://us.download.nvidia.com/tesla/528.33/528.33-data-center-tesla-desktop-win10-win11-64bit-dch-international.exe",
+          "PackageName": "nvidiaDrivers",
+          "Extract": true
+      }
+  },
+  {
+      "Type": "CudaAndNvidiaGPUDriverInstallation",
+      "Parameters": {
+          "Scenario": "InstallCudaAndNvidiaGPUDriverForWindows",
+          "WinCudaDriversPackageDownloadedFromBlob": false,
+          "RebootRequired": false,
+          "WinCommandLineArgs": "-y -s",
+          "PackageName": "nvidiaDrivers"  
+      }
+  }
+  ```
+  </div>
+  
+### Windows example for downloading drivers from Web
+
+  <div class="code-section">
+
+  ```json
+{
+      "Type": "DependencyPackageInstallation",
+      "Parameters": {
+          "Scenario": "DownloadCudaAndNvidiaDriverFromBlob",
+          "BlobContainer": "packages",
+          "BlobName": "<package-name-in-blob>",
+          "PackageName": "nvidiaDrivers",
+          "Extract": true
+      }
+  },
+  {
+      "Type": "CudaAndNvidiaGPUDriverInstallation",
+      "Parameters": {
+          "Scenario": "InstallCudaAndNvidiaGPUDriverForWindows",
+          "WinCudaDriversPackageDownloadedFromBlob": true,
+          "RebootRequired": false,
+          "WinCommandLineArgs": "-y -s",
```

Comment:
-y -s seems pretty standard, would there be any other reasonable options? If not let's remove this parameter

Reply:
removed the parameter